### ISSUE TITLE
Documentation Request: Working Example of Page Redirect instead of Pop-Up for Angular

### ISSUE THREAD

**Issue by akleimrey** [AzureAD/microsoft-authentication-library-for-js#2197]
Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
- [ ] `msal@1.x.x` or `@azure/msal@1.x.x`
- [ ] `@azure/msal-browser@2.x.x`
- [ ] `@azure/msal-angular@0.x.x`
- [x] `@azure/msal-angular@1.x.x`
- [ ] `@azure/msal-angularjs@1.x.x`

## Documentation location
- [ ] docs.microsoft.com
- [ ] MSAL.js Github Wiki
- [ ] README file
- [ ] Other (please fill in)
- [x] Documentation does not exist

## Description
Basically what the title says. While it initially appears to be JUST a flag, the flow of the application is entirely different and not explained in the setup so far.

**Comment by jasonnutter** [comment]
@akleimrey Our MSAL Angular samples demonstrate how to do popup and redirect based authentication: https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-samples

Is there something specific you would like more clarification on? Thanks!

**Comment by akleimrey** [comment]
@jasonnutter Works for my case! It was less of a coding issue than a configuration one. Redirect works much better than pop up on almost all fronts.
